app/views/static/cloud_tenant/aggregate-tenant-status-card.html.haml

```diff
@@ -0,0 +1,7 @@
+.aggregate_status{"ng-controller" => "aggregateTenantStatusCardController as vm"}
```

Comment:
Do we really need a third aggregate status card component?

There should only be one, please try to make sure either `ems_common/aggregate-status-card` or `ems_container/_aggregate-status-card` work for you and use one of those.

Reply:
(Also, seems like nothing is using this file, as `%aggregate-status-card` always uses `ems_common/aggregate-status-card`.)